pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -58,7 +60,8 @@ func (c *component) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContex
 
 // RenderManifests renders the Helm chart templates with values provided.
 func (c *component) RenderManifests() (map[string]string, error) {
-	helmChart, err := util.LoadChartFromAssets(fmt.Sprintf("/components/%s/manifests", name))
+	p := filepath.Join(assets.ComponentsSource, name)
+	helmChart, err := util.LoadChartFromAssets(p)
```

Comment:
It seems to me, that we should add a helper method to `components`, let's call it `ComponentChart()`, which would wrap `util.LoadChartFromAssets`. This way, we don't need 2 more imports and we have even simpler API than before:
```go
helmChart, err := components.ComponentChart(name)
```

Reply:
I can look into it. `components.ComponentChart()` is stuttering though - I'd use `components.Chart()`.

Reply:
Done.